src/silo/query_engine/filter_expression/nucleotide_symbol_equals.cpp

```diff
@@ -0,0 +1,48 @@
+#include "silo/query_engine/filter_expressions/nucleotide_symbol_equals.h"
+
+#include <vector>
+
+#include "silo/query_engine/operators/bitmap_selection.h"
+#include "silo/query_engine/operators/complement.h"
+#include "silo/query_engine/operators/index_scan.h"
+
+#include "silo/storage/database_partition.h"
+
+namespace silo::query_engine::filter_expressions {
+
+NucleotideSymbolEquals::NucleotideSymbolEquals(unsigned int position, silo::NUCLEOTIDE_SYMBOL value)
+    : position(position),
+      value(value) {}
+
+std::string NucleotideSymbolEquals::toString(const silo::Database& /*database*/) {
+   std::string res = std::to_string(position) + genomeSymbolRepresentation(value);
```

Comment:
```
[2023-05-25 17:35:11.905] [console_logger] [info] [query_handler.cpp:47] received query: {"action":{"type":"Aggregated"},"filterExpression":{"children":[{"children":[{"position":1,"symbol":"-","type":"NucleotideEquals"},{"position":23062,"symbol":"T","type":"NucleotideEquals"}],"type":"And"}],"type":"And"}}
[2023-05-25 17:35:11.905] [console_logger] [debug] [query_engine.cpp:57] Parsed query: ((0- & 23061T))
```
This is confusing. Maybe let's add one again here? We should think of a better solution here. I'm pretty sure that this will cause a bug in the long run.
```suggestion
   std::string res = std::to_string(position + 1) + genomeSymbolRepresentation(value);
```